- Adding workSection

sonuengineer opened the pull request:
- Adding workSection
- Adding AboutUs section
- Adding Skills

sonuengineer committed:
Update index.html

sonuengineer committed:
Update style.css

sonuengineer committed:
Adding work_section &About-us_section

sonuengineer committed:
uploading images

bienvenuushindi commented on index.html:
```diff
@@ -1,48 +1,285 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style1.css" />
```
:red_circle: No styling is applied to your HTML code. I think you got the file name wrong. Please make sure to fix the misspelled file :bug: 
![Peek 2022-11-02 18-57](https://user-images.githubusercontent.com/26736582/199553235-bf2e8593-a84d-45db-8581-8e0dcf0fd692.gif)

Please make sure You stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

bienvenuushindi reviewed (changes requested):
### Hi @sonuengineer ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

sonuengineer committed:
Update index.html file

Alfredbis29 commented on index.html:
```diff
@@ -1,48 +1,285 @@
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
```
- [ ] Good job on the changes you have made, but there is still room for improvement if you want your project to be completed. Please use the correct path to display the card and this section because it is empty and only one section is showing. Please fix this for project to be completed :point_down: 
![empty](https://user-images.githubusercontent.com/88894525/199560110-b53642dd-d037-4575-9d51-63130aa977b9.png)

Alfredbis29 reviewed (changes requested):
Hi @sonuengineer , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### REQUEST CHANGES :black_circle: 
 ## Highlights
:white_check_mark:  Linters
:white_check_mark: Descriptive PR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sonuengineer commented:
I upload all the content to index.html

On Wed, Nov 2, 2022 at 11:07 PM Alfred Bisimwa ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @sonuengineer <https://github.com/sonuengineer> ,
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
> REQUEST CHANGES ⚫ Highlights
>
> ✅ Corect Gitflow
> ✅ Good description commit
> ✅ Linters
> ✅ Descriptive PR
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/sonuengineer/MyPortfolio/pull/4#discussion_r1012096544>
> :
>
> > +      <div class="card-body">
>
> +        <h1 class="card-title">Profesional Art Printing Data</h1>
>
> +        <p class="supporting-text">
>
> +          A daily selection of privately personalized reads; no accounts or
>
> +          sign-ups required. has been the industry's standard
>
> +        </p>
>
> +        <ul class="card-ul">
>
> +          <li class="card-li">
>
> +            <a href="" class="card-a">html</a>
>
> +          </li>
>
> +
>
> +          <li class="card-li">
>
> +            <a href="" class="card-a">bootstrap</a>
>
> +          </li>
>
> +
>
>
>
>    - Good job on the changes you have made, but there is still room for
>    improvement if you want your project to be completed. Please use the
>    correct path to display the card and this section because it is empty and
>    only one section is showing. Please fix this for project to be completed
>    👇
>    [image: empty]
>    <https://user-images.githubusercontent.com/88894525/199560110-b53642dd-d037-4575-9d51-63130aa977b9.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sonuengineer/MyPortfolio/pull/4#pullrequestreview-1165722978>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APC2EVRCUFECHY2GSWGEHSTWGKRHFANCNFSM6AAAAAARVH5KLM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sonuengineer committed:
uploading images

Alfredbis29 commented:
Hello @sonuengineer , I hope you are doing fine , I just find that issue after you updated that file , is all sections displayed on your hand?

sonuengineer committed:
update index.html

sonuengineer committed:
update style.css

sonuengineer commented:
> Hello @sonuengineer , I hope you are doing fine , I just find that issue after you updated that file , is all sections displayed on your hand?

@Alfredbis29 yes

sonuengineer commented:
@Alfredbis29  can you chack now

sonuengineer commented:
Yes, it is.

On Wed, Nov 2, 2022 at 11:41 PM Alfred Bisimwa ***@***.***>
wrote:

> Hello @sonuengineer <https://github.com/sonuengineer> , I hope you are
> doing fine , I just find that issue after you updated that file , is all
> sections displayed on your hand?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sonuengineer/MyPortfolio/pull/4#issuecomment-1301033580>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APC2EVURBYUZ7EXEDYAOTYLWGKVETANCNFSM6AAAAAARVH5KLM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sonuengineer commented:
@Alfredbis29 can you tell me what I have to do ,i check three times everything is good from my side

Alfredbis29 commented:
If you are ready you can ask for the other review and I hope you get approved, good luck 👍

ha-manel commented on index.html:
```diff
@@ -1,48 +1,285 @@
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
```
- [x] Just like for the first reviewer asked, the design should be as close as possible to the design in Figma, for that you can increase the `height` of the `see project` button 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199577446-2d29b30f-5743-4055-98e0-4b1c00ed6b92.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199577842-ee576b50-979a-4a66-910a-930f641fccf5.png)

GiuseppeTG replied:
- [ ] As the previous reviewers asked, please make the button of your project card match the style of the Figma template. 🤓 

Figma:
![Captura de Pantalla 2022-11-02 a la(s) 2 58 57 p  m](https://user-images.githubusercontent.com/86022690/199591435-3eaaad82-8624-411a-84b6-ff2b7bfd4ee0.png)

Yours:
![Captura de Pantalla 2022-11-02 a la(s) 2 59 11 p  m](https://user-images.githubusercontent.com/86022690/199591479-5631daf8-585c-4d5b-8dd0-86171ca0a45d.png)

Please also use the capitalized letters as in the Figma design 👍

Qoosim replied:
- [ ] From `375px` view port and above, your `See project` button should look exactly like what your have on the Figma template. Anything below `375px` view port can be ignored. But in your own case, from `375px` to `398px` view port, this is how the button appears on mobile. :point_down: 

![btn-responsiveness](https://user-images.githubusercontent.com/26597432/200126083-1f6175ea-3eb0-43c1-9250-2be6dfffd9e6.png)

- From `399px` view port above, your `See Project` button appears as expected.

![from-another-vp](https://user-images.githubusercontent.com/26597432/200126272-a5e0c813-6bca-4ff3-80cd-5e070c4d8a20.png)

- [ ] Kindly make it appear as we have it on the Figma template from `375px` view port and above.

ha-manel commented on index.html:
```diff
@@ -1,48 +1,285 @@
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
```
- [x] As requested by the previous reviewer, kindly follow the design in Figma by updating the `font-family` of this paragraph to the one in Figma (`Inter`)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199578174-a7de572d-1ee8-413c-823d-e1c519a27bb7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199578209-af74cc94-3e1e-444f-aaa1-61cd799cf478.png)

ha-manel commented on index.html:
```diff
@@ -1,48 +1,285 @@
+
+
+    <section class="work-section">
+      <h1>My Recent Works</h1>
```
[Optional]
I'm only marking this as optional since the previous reviewer missed this detail, however, it should be fixed to have a semantically correct project ✨ 
- [ ] There should only be one `h1` in each page which is the main header of the page, so for better semantics kindly change this `h1` and any other following `h1` tag to another `hx` tag 👍

ha-manel commented on index.html:
```diff
@@ -1,48 +1,285 @@
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul>
+          <li>
+            <a href="#">css</a>
+          </li>
+          <li>
+            <a href="#">html</a>
+          </li>
+          <li>
+            <a href="#">Bootstraps</a>
+          </li>
+          <li>
+            <a href="#">Ruby</a>
```
[Optional]
- [ ] Since this is a normal list of technologies and not links, the use of anchor tags `a` here is wrong. It's better if you leave as normal text in `li` 👍

ha-manel commented on index.html:
```diff
@@ -1,48 +1,285 @@
+        </h3>
+        <ul class="lang-ul">
+          <li class="lang-li">
+            <h5 class="lang-p">javascript</h5>
```
[Optional]
- [ ] It's better if technologies/languages/tools names are capitalized 👍

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi @sonuengineer 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media3.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sonuengineer committed:
update style.css

sonuengineer committed:
update style.css

sonuengineer committed:
update style.css

sonuengineer committed:
update style.css

GiuseppeTG commented on index.html:
```diff
@@ -1,48 +1,285 @@
+            <a href="#">Ruby</a>
+          </li>
+        </ul>
+        <button>see project</button>
```
- [OPTIONAL] This button has a margin that I think can be removed so that it is aligned as in the Figma design. I marked this as optional because it was not asked by the previous reviewer but it should have been implemented before 😅 

![Captura de Pantalla 2022-11-02 a la(s) 3 10 17 p  m](https://user-images.githubusercontent.com/86022690/199593020-38c2849c-8ad0-42f2-a71c-889a31b5bf3e.png)

GiuseppeTG reviewed (changes requested):
Hi @sonuengineer 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![Alt Text](https://media0.giphy.com/media/wA6aYAO3jH6RmUIBYs/giphy.gif)

 ## Highlights
✅ Good job fixing some of the previous issues
✅ Linters passing 
✅ Everything is correctly displayed now

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sonuengineer committed:
update style.css

i changed the see project button which was asked by the reviewer and I changed according to figma style as welll.

sonuengineer committed:
update style.css

sonuengineer commented:
I change the see project button similar to Figma which is having height of 48px and the output is the same as expected

![image](https://user-images.githubusercontent.com/63283798/200124513-366812eb-4d51-44d5-9191-8eb871df5942.png)

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle:

Hi @sonuengineer, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://user-images.githubusercontent.com/33840476/192236577-e2d71173-44d8-4ac8-9615-bf78ba730792.gif)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] Kindly attend to the margin part of this  too :point_down: 

![btn-space](https://user-images.githubusercontent.com/26597432/200126431-62edea2c-e53b-41c6-af45-d84db3171461.png)

sonuengineer committed:
Create imgs

sonuengineer committed:
Delete imgs

sonuengineer committed:
Create s.png

sonuengineer committed:
Add files via upload

sonuengineer committed:
Create styles.css

sonuengineer committed:
Create normalize.css

sonuengineer committed:
Update index.html

sonuengineer committed:
Update styles.css

ElsonOtake reviewed (approved):
# status: APPROVED :green_circle: :100: 

Hi @sonuengineer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

![Minions](https://user-images.githubusercontent.com/85113567/164945882-0a9554ea-956c-4fa0-a95c-379285283c1d.gif)

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
